test/rules/no-internal-module/test.ts.lint

```diff
@@ -2,10 +2,14 @@ namespace foo {
 }
 
 module bar {
+~~~~~~~~~~~~
 }
+~ [0]
```

Comment:
this doesn't look great; we should probably change this rule to report errors just on the `module bar` part: #939